### ISSUE TITLE
Codegen for TypeScript should quote primitive types within array/map decorators

### ISSUE THREAD

**Issue by ResonAtom** [colyseus/schema#47]
```
export class MyState extends Schema {
  @type(["int16"])
  terrainArray:ArraySchema<number>
```

> npx schema-codegen WorldRoom.ts --output ../fe-client/src --ts

The generated MyState.ts file has the int16 without quotes:
```
@type([ int16 ]) public terrainArray: ArraySchema<number> = new ArraySchema<number>();
```

Causing "Cannot find name 'int16'."  My purpose here is just to codegen my server's classes for use in my separate TypeScript client for vscode hinting.

I'm new but I'll submit a PR in a minute.

**Comment by endel** [comment]
Thanks a lot for your PR @ResonAtom! I've just published `@colyseus/schema@0.5.6` with your fix!

**Comment by ResonAtom** [comment]
@endel I've been spending a few hours daily with Colyseus and it's enjoyable to work with, thank you!
